aws-redshiftserverless-namespace/src/main/java/software/amazon/redshiftserverless/namespace/DeleteHandler.java

```diff
@@ -32,6 +32,14 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                             .handleError(this::defaultErrorHandler)
                             .done(deleteNamespaceResponse -> {
                                 logger.log(String.format("%s %s deleted.",ResourceModel.TYPE_NAME, model.getNamespaceName()));
+                                // TODO: Need to add a stabilize operation to verify if secret is deleted
+                                // This is a temporary fix to handle deletion of secrets for managed passwords
+                                // Since deletion of secret is handled async CTv2 is failing even in SingleTestMode
+                                try {
+                                    Thread.sleep(10000);
```

Comment:
can we add this only if the secret exists? I mean, can we do a check for the secret before the wait?

Reply:
For secret exists we do need the FAS policy changes to be done. That changes will be done by @nidhimanthale